3600/3/app/controllers/app_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'sinatra'
+require 'rubygems'
+require 'sinatra/activerecord'
+require 'erb'
+require 'sinatra/flash'
+require 'dotenv'
+require 'require_all'
+require_rel '../models/'
+
+require_relative 'restaurant_controller'
+require_relative 'user_controller'
+
+class AppController < Sinatra::Base
+  Dotenv.load
+  configure do
+    enable :sessions
+    set :public_folder, File.dirname(__FILE__) + '/public'
+    set :session_secret, ENV['key']
+  end
+
+  use UserController
+  use RestaurantController
```

Comment:
Put this two lines to `config.ru` file

Reply:
Before `run AppController`

Reply:
Done

Reply:
Done

Reply:
Done

Reply:
![Screenshot from 2019-08-02 22-53-25](https://user-images.githubusercontent.com/46228563/62395254-5d728500-b578-11e9-8c06-b61611e73737.png)